docs/guides/writing_stubs.rst

```diff
@@ -106,3 +106,478 @@ particularly with false positives.
 If your package has some particularly complex aspects, you could even consider
 writing dedicated typing tests for tricky definitions. For more details, see
 :ref:`testing`.
+
+Stub Content
+============
+
+This section documents best practices on what elements to include or
+leave out of stub files.
+
+Modules excluded fom stubs
+--------------------------
+
+Not all modules should be included into stubs.
```

Comment:
Not a native speaker, but "into stubs" sounds wrong to be. "in stubs"?

Reply:
Fixed, thanks!